content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -179,15 +181,31 @@ python3 -m datatrails_scitt_samples.scripts.create_hashed_signed_statement \
     ```bash
     python3 -m datatrails_scitt_samples.scripts.verify_receipt \
       --transparent-statement-file $TRANSPARENT_STATEMENT_FILE \
-      --leaf $LEAF
+      --leaf $(jq -r .leaf <<<"$RESPONSE")
     ```
 
-    Following the example above, $LEAF should be:
+    The verification should pass with:
 
     ```output
-    30f5650fbe3355ca892094a3fbe88e5fa3a9ae47fe3d0bbace348181eb2b76db
+    verification passed
     ```
 
+1. Simulate a failed verification, by altering the `.leaf` value
+
+    ```bash
+
+    python3 -m datatrails_scitt_samples.scripts.verify_receipt \
+      --transparent-statement-file $TRANSPARENT_STATEMENT_FILE \
+      --leaf $(jq -r .leaf <<<"$RESPONSE")"-foo"
```

Comment:
The leaf needs to be a valid hex value of 0x followed by 64 hex characters. In total representing 32 byte

Reply:
You could just link to the test case that does this with real event data

https://github.com/datatrails/datatrails-scitt-samples/blob/main/tests/test_verify_receipt.py#L52

as a developer I would find that example more useful